src/LogHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Text.RegularExpressions;
+
+namespace ModelRepoBrowser;
+
+public static partial class LogHelper
+{
+    [GeneratedRegex(@"\r\n|\r|\n")]
+    private static partial Regex NewLinesRegex();
+
+    public static string? Escape(string? input)
+    {
+        return input == null ? null : NewLinesRegex().Replace(input, " ");
+    }
+}
```

Comment:
REQ: Da es sich um eine String-Manipulation handelt, welche auch ausserhalb des Loggings aufgerufen werden kann, sollte die Methode als Extension auf `string?` in einer `StringExtensions`-Klasse implementiert werden. Das macht das Ganze verständlicher und wir führen dann keine ominöse `Helper`-Klasse. 

REQ: XML-Doku hinzufügen.